src/Ledger/TokenAlgebra.lagda

```diff
@@ -53,5 +54,6 @@ record TokenAlgebra : Set₁ where
   sumᵛ : List Value → Value
   sumᵛ = foldr _+ᵛ_ (inject 0)
 \end{code}
-\caption{Token algebras, used for multi-assets}
+\end{AgdaAlign}
+\caption{Token algebras, used for multi-assets; \AgdaBound{PolicyId} specifies a type (e.g., with natural numbers) used to represent policy identifiers.}
```

Comment:
We already have a pattern for this, see the figure right below (Definitions used in Addresses). Maybe that pattern isn't that great (if you don't like it, make an issue to change it everywhere), but this should still follow it otherwise it's confusing (is `PolicyID` not an abstract type?).

Reply:
Okay, I'll check out the existing pattern and conform.

No, `PolicyID : Type` is a parameter passed into this (the `TokenAlgebra`) module.